sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/Transforms.java

```diff
@@ -282,8 +281,21 @@ private static FormField setFormField(String name, FieldData valueData, FieldVal
             default:
                 throw LOGGER.logExceptionAsError(new RuntimeException("FieldValue Type not supported"));
         }
-        return new FormField(name, null, valueData, value,
-            setDefaultConfidenceValue(fieldValue.getConfidence()));
+
+        FieldData labelData = null;
+        // if 'name' is same as the raw field value text, it means this FormField is not a label.
+        // Otherwise, the FormField is for label, which might have label data, such as bounding box values.
+        if (name != null && name.equals(fieldValue.getText())) {
```

Comment:
Not sure I follow. How will this affect what we already shipped?

Reply:
it was confused. Updated to only adding valueData for non-ARRAY type when valueData is null